test/services/cache/ExportsCacheModelBuilder.scala

```diff
@@ -117,6 +117,20 @@ trait ExportsCacheModelBuilder {
         BorderTransport(borderModeOfTransportCode, meansOfTransportOnDepartureType, meansOfTransportOnDepartureIDNumber)
       )
     )
+  def withConsignmentReferences(consignmentReferences: Option[ConsignmentReferences]): CacheModifier =
+    _.copy(consignmentReferences = consignmentReferences)
+
+  def withLocations(locations: Locations): CacheModifier =
+    _.copy(locations = locations)
+
+  def withParties(parties: Parties): CacheModifier =
```

Comment:
No need for this when you can do 
```
withExporterDetails(...),
withConsigneeDetails(...),
withDeclarantDetails(...)
```
etc

Reply:
these didn;t exist so have added them to the builder